README.md

```diff
@@ -1,4 +1,7 @@
 # bun
+<p align="center">
```

Comment:
Prefer: `[![](https://bun.sh/logo@2x.png)](https://bun.sh)`. 

* Adding HTML to the Markdown ruins the lightweight syntax and centering it adds nothing but noise to the plaintext
* `alt="Logo"` is not helping assistive technology. The logo is purely decorative so it it should be empty (e.g. `alt=""`)